geostore/urls.py

```diff
@@ -1,69 +1,45 @@
-from django.conf import settings
 from django.http import HttpResponseNotFound
-from django.urls import path
-from drf_yasg import openapi
-from drf_yasg.views import get_schema_view
-from rest_framework import permissions, routers
+from django.urls import path, include
+from rest_framework import routers
 
 from .tiles.views import (LayerGroupTileDetailView, LayerTileDetailView,
                           MultipleTileJsonView, TileJsonView)
 from .views import (FeatureRelationViewSet, FeatureViewSet,
                     LayerRelationViewSet, LayerViewSet)
-from . import settings as app_settings
+
 
 app_name = 'geostore'
 
-schema_view = get_schema_view(
-    openapi.Info(
-        title="Terra PI",
-        default_version='v1',
-        description="The future of Makina Corpus",
-    ),
-    public=True,
-    permission_classes=(permissions.AllowAny, ),
-)
+router = routers.SimpleRouter()
+
+router.register(r'layer', LayerViewSet)
+router.register(r'layer/(?P<layer>[\d\w\-_]+)/feature', FeatureViewSet,
+                base_name='feature')
+router.register(r'layer_relation', LayerRelationViewSet)
+router.register(r'layer_relation/(?P<layerrelation_pk>\d+)/feature_relation',
+                FeatureRelationViewSet)
+
 
 urlpatterns = [
-    path(r'group/<str:slug>/tilejson',
+    path('', include(router.urls)),
+    path('group/<str:slug>/tilejson',
          MultipleTileJsonView.as_view(),
          name='group-tilejson'),
-    path(r'layer/<int:pk>/tilejson',
+    path('layer/<int:pk>/tilejson',
          TileJsonView.as_view(),
          name='layer-tilejson'),
-    path(r'group/<str:slug>/tiles/<int:z>/<int:x>/<int:y>/',
+    path('group/<str:slug>/tiles/<int:z>/<int:x>/<int:y>/',
          LayerGroupTileDetailView.as_view(),
          name='group-tiles'),
     # Fake pattern to be able to reverse this
-    path(r'group/<str:slug>/tiles/{z}/{x}/{y}/',
+    path('group/<str:slug>/tiles/{z}/{x}/{y}/',
          lambda request, **kwargs: HttpResponseNotFound(),
          name='group-tiles-pattern'),
-    path(r'layer/<int:pk>/tiles/<int:z>/<int:x>/<int:y>/',
+    path('layer/<int:pk>/tiles/<int:z>/<int:x>/<int:y>/',
          LayerTileDetailView.as_view(),
          name='layer-tiles'),
     # Fake pattern to be able to reverse this
-    path(r'layer/<int:pk>/tiles/{z}/{x}/{y}/',
+    path('layer/<int:pk>/tiles/{z}/{x}/{y}/',
          lambda request, **kwargs: HttpResponseNotFound(),
          name='layer-tiles-pattern'),
 ]
-
-router = routers.SimpleRouter()
-
-router.register(r'layer', LayerViewSet)
-router.register(r'layer/(?P<layer>[\d\w\-_]+)/feature', FeatureViewSet,
-                base_name='feature')
-router.register(r'layer_relation', LayerRelationViewSet)
-router.register(r'layer_relation/(?P<layerrelation_pk>\d+)/feature_relation',
-                FeatureRelationViewSet)
-
-urlpatterns += router.urls
```

Comment:
Why delete this ?

Reply:
best practice is to use include() instead of +=

Reply:
path('', include(router.urls)),